readme-template.md

```diff
@@ -0,0 +1,92 @@
+# Repo Title
+
+<!-- Not all of these are essential but keep this in mind for the repo you are using
+![Build badge]() ![Test Coverage]() ![npm package]() -->
+
+<!--Short sentence on what the repo does-->
+
+##Contents
+* [Further Documentation](#further-documentation)
+* [System Overview](#system-overview)
+* [Installation](#installation)
+* [Running Locally](#running-locally)
+* [Testing](#testing)
+	* [Automated Testing](#automated)
+	* [Manual testing](#manual)
+* [Script cheat sheet](#cheat-sheet)
+* [Built With](#built-with)
+* [Linting](#linting)
+* [Deploying](#deploying)
+* [Contributing Rules](#contributing-rules)
+* [Common Questions](#common-questions)
+
+## Further Documentation
+
+<!-- Examples of some documentation to link from the readme -->
+* **[System Roadmap]()**
+* **[Regression Testing Checklist]()**
+* **[Change Log]()**
+
+Include any other relevant documentation in this section.
+
+## System Overview
+
+* More in-depth overview of the system, what this is trying to accomplish and how it achieves its goal.
+* Keep it concise, write this section like the first paragraph of a blog post, what is the point of this repo?
+* Perhaps label some of the core tech used in the repo and link to the external websites/documentation
+
+## Installation
+
+* Step-by-step guide on installation
+
+## Running Locally
+
+* Step-by-step guide of setting up locally
+
+## Script Cheat Sheet
+
+* List scripts used in the repo and a description of what they do
+* Example: `npm run build` := Generates `dist` folder with changes made in `lib`
+
+## Testing
+
+* What CI do the test run on?
+* What testing framework is used?
+* How are the tests structured in this repo? Is there a strict writing style to follow?
+
+### Automated Testing
+
+* Step-by-step to run all automated testing.
+* What are some of the commands I need to know when running test in my local environment?
+* Any tips on running singular tests or groups of tests at once in this repo?
+
+### Manual Testing
+
+* Link to external regression checklist document
+* If anything particular is setup for this repo, describe how you would manually test a feature branch before going to production
+
+## Built With
+
+* What frameworks are used? Links to framework API docs.
+* Essential node modules used, links to documentation/repo.
+
+## Linting
+
+* How to I lint specific file types?
```

Comment:
Perhaps, sections like this should be considered on a repo by repo basis. Some might be straight forward to link to the config file. But some of our repos for instance vary in linting rules so makes sense to highlight the differences here